From b35fa48be6b65ca6bb8f185ab0774dd351b81eba Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Sun, 21 Feb 2016 15:32:07 +0200
Subject: [PATCH 0067/1345] dts: a7040: re-enable AP806 I2C bus for a7040 RZ
 DB board

commit  db8a843ef5c7f74ec894c21f279b81496599ac49 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Apparantly the AP806 I2C is used and not the CP110

Change-Id: I7e276cbe866cf9d474fd18ea2ca50bccf7901b1d
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040-rz-db.dtsi |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040-rz-db.dtsi b/arch/arm64/boot/dts/marvell/armada-7040-rz-db.dtsi
index 4c322d3..d1a0ba2 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040-rz-db.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040-rz-db.dtsi
@@ -61,6 +61,10 @@
 			serial@512000 {
 				status = "okay";
 			};
+			i2c@511000 {
+				status = "okay";
+				clock-frequency = <100000>;
+			};
 		};
 	};
 
-- 
1.7.9.5

